drivers/segger/sysview.c

```diff
@@ -406,7 +420,7 @@ void sched_note_syscall_enter(int nr, int argc, ...)
   SEGGER_SYSVIEW_MarkStart(nr);
 }
 
-void sched_note_syscall_leave(int nr, uintptr_t result)
+static void sysview_syscall_leave(FAR struct note_driver_s *drv, int nr)
 {
   nr -= CONFIG_SYS_RESERVED;
 
```

Comment:
remove sysview_isenabled_syscall and NOTE_FILTER_SYSCALLMASK_ISSET, already check in common code